iotile_analytics_core/iotile_analytics/core/channels/cloud_api.py

```diff
@@ -120,6 +120,7 @@ def fetch_events(self, slug):
 
             for i, event in enumerate(data):
                 extra_data[i]['event_id'] = event['id']
+                extra_data[i]['has_raw_data'] = event['has_raw_data'] if 'has_raw_data' in event else False
```

Comment:
Random trick learned from @diametralpitch, you can simplify this to:

```
event.get('has_raw_data', False)
```

get never raises a `KeyError` and you can specify an optional default instead of None as the second parameter